src/Model/PhasorDynamics/Exciter/IEEET1/Ieeet1Impl.hpp

```diff
@@ -131,7 +142,7 @@ namespace GridKit
         // other variables.
         if (signals_.template isAssigned<Ieeet1InternalVariables::EFD>())
         {
-          efd0 = y_[7]; //<- generator needs to be initialized first
+          efd0 = y_[7]; ///<- generator needs to be initialized first
         }
```

Comment:
I think this will confuse doxygen and create a weird documentation item. Syntax `///<` is intended to add doc entries for variable definitions on the same line. I'm not sure what it will do for the assignment.

Reply:
Addressed in [2b0573e](https://github.com/ORNL/GridKit/pull/232/commits/2b0573e630404a68fe52f7cdb6036c5bfb2bfe1b)